package.json

```diff
@@ -101,6 +103,9 @@
     "storybook:static:build": "build-storybook --config-dir config/storybook --output-dir build/storybook",
     "storybook:static:serve": "serve build/storybook --port 5555 || true",
     "storybook:static:stop": "pkill -f \"node_modules/.bin/serve build/storybook\"",
+    "svg:svgo": "svgo --folder=./assets/Icons/svgs-from-design --output=./build/icons --config=./.svgo.yml",
```

Comment:
is there a reason we don't just have designers dump the SVGs directly into `assets/Icons` and pull from there?

Reply:
since we're doing a full build of the assets/Icons directory each time i think we should also be cleaning the `build/icons` and `src/components/Icon/icons` directories to ensure we're not accidentally keeping any artifacts:

```yarn add -D del-cli```

```
"svg:clean-build": "rimraf build/icons",
"svg:clean-components": "del 'src/components/Icon/icons/*.tsx'",
```

Reply:
We already have `rimraf`, no need for `del-cli`.

Reply:
we need to remove all the component .tsx files but leave the index.ts file, i think rimraf just deletes directories. unless we want to generate the index.ts file as well, that would mean one less manual step of adding new icon imports

Reply:
`rimraf` supports globs fine. No need for `del-cli`.

Reply:
oh nice i was not aware